astropy_timeseries/sampled.py

```diff
@@ -199,3 +200,56 @@ def to_pandas(self):
             df = Table(self).to_pandas(index='time')
 
         return df
+
+    @classmethod
+    def read(self, filename, time_column, time_format=None, time_scale=None, format=None, *args, **kwargs):
```

Comment:
``time_column`` should be optional since it isn't needed for e.g. Kepler files.

Reply:
Should be fixed now.